staging/src/k8s.io/cli-runtime/pkg/resource/kustomizevisitor_test.go

```diff
@@ -104,7 +104,7 @@ metadata:
     note: This is a test annotation
   labels:
     app: nginx
-  name: foo-secret-bar-9btc7bt4kb
+  name: foo-secret-bar-82c2g5f8f6
```

Comment:
same question about the generated suffix changing

Reply:
same thing happening here